Marlin/src/HAL/NATIVE_SIM/sim/visualisation.h

```diff
@@ -212,7 +212,7 @@ class Visualisation {
   bool input_state[6] = {};
   glm::vec<2, int> mouse_lock_pos;
 
-  std::array<GLfloat, 24 * 10> g_vertex_buffer_data{
+  const std::array<GLfloat, 24 * 10> g_vertex_buffer_data{
```

Comment:
This can't be const! It's changed in the code, because it draw the bed and nozzle.